test/DynamoCoreTests/CrashProtectionTests.cs

```diff
@@ -147,5 +147,17 @@ public void FlagIsSetToTrueAfterRunSuccessfullyCompletes()
             }
         }
 
+        [Test]
+        public void MAGN_7146()
```

Comment:
@monikaprabhu: Thanks for adding. Just note that this behavior will work for any file with the Manula Flag set, not just VM related files